x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/schemas.ts

```diff
@@ -7,6 +7,7 @@
 import Joi from 'joi';
 
 /* eslint-disable @typescript-eslint/camelcase */
+export const anomaly_threshold = Joi.number();
```

Comment:
You'll want this to keep people from abusing this field and adding either a float or something less than 0 or greater than 100:

```ts
  .integer()
  .greater(-1)
  .less(101);
```